src/aes128gcm.rs

```diff
@@ -14,7 +14,7 @@ const ECE_AES128GCM_HEADER_LENGTH: usize = 21;
 // The max AES128GCM Key ID Length is 255 octets. We use far less of that because we use
 // the "key_id" to store the exchanged public key since we don't cache the key_ids.
 // Code fails if the key_id is not a public key length field.
-const ECE_AES128GCM_PAD_SIZE: usize = 1;
+pub(crate) const ECE_AES128GCM_PAD_SIZE: usize = 1;
```

Comment:
part of me *REALLY* wants this to be just `ECE_PAD_SIZE` and kinda force the point by requiring the package if you want to use both (e.g. `aes128gcm.ECE_PAD_SIZE`), but that might lead to unintended bugs.